pkg/apis/direct.csi.min.io/v1alpha1/deepcopy_generated.go

```diff
@@ -1,19 +1,19 @@
 //go:build !ignore_autogenerated
 // +build !ignore_autogenerated
 
-// This file is part of MinIO Direct CSI
-// Copyright (c) 2021 MinIO, Inc.
-//
+// This file is part of MinIO DirectPV
+// Copyright (c) 2022 MinIO, Inc.
+
 // This program is free software: you can redistribute it and/or modify
 // it under the terms of the GNU Affero General Public License as published by
 // the Free Software Foundation, either version 3 of the License, or
 // (at your option) any later version.
-//
+
```

Comment:
Are you regenerated the code with new header having updated name and year or editing manually?